modules/rostests/apitests/shell32/ShellExecuteEx.cpp

```diff
@@ -111,15 +111,37 @@ getCommandLineFromProcess(HANDLE hProcess)
     PEB peb;
     PROCESS_BASIC_INFORMATION info;
     RTL_USER_PROCESS_PARAMETERS Params;
+    LONG status;
```

Comment:
```suggestion
    NTSTATUS status;
```
(also our code style uses `Status` capitalized that way for status variables)

Reply:
Fixed.